### PR TITLE
Migrate to ROS 2

### PR DESCRIPTION
**Pull request by luator**
## Description

Migrate to ROS 2

## How I Tested

Built the BLMC_EI project and ran a few demos.

### CODE REVIEW

**Comment by luator** [comment]
@shrutij01 FYI

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,63 +1,86 @@
...
+        scripts/check_position_control_accuracy.py
+        scripts/evaluate_policy.py
+        scripts/profiling.py
+        scripts/pybullet_backend.py
+        scripts/replay_action_log.py
+        scripts/rrc_evaluate
+        scripts/run_evaluate_policy_all_levels.py
+        scripts/run_replay_all_levels.py
+    DESTINATION lib/${PROJECT_NAME}
+)
```
Comment (MaximilienNaveau) [inline CMakeLists.txt:53]: idem install files and rename + add exec

Comment (luator) [reply CMakeLists.txt:53]: done

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,63 +1,86 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
```
Comment (MaximilienNaveau) [inline CMakeLists.txt:1]: the version on ubunut18.04 is 3.10 btw, it does change anything really but just for you to know.
This is why I have put 3.10 everywhere.

**Review by MaximilienNaveau — APPROVED** [review]

**Pull request merged**